networking.tf

```diff
@@ -0,0 +1,161 @@
+resource "aws_vpc" "vpc" {
+  cidr_block = local.cidr_block
+  tags = {
+    Name = var.name
```

Comment:
use name prefix. Or else when we cannot use this module more than once, as the resources have same name.